docs/source/install.rst

```diff
@@ -98,6 +98,32 @@ Please make sure that only one CuPy package (``cupy`` or ``cupy-cudaXX`` where X
   $ pip freeze | grep cupy
 
 
+Install CuPy from conda-forge
+-----------------------------
+
+Conda/Anaconda is a cross-platform package management solution widely used in scientific computing and other fields.
+The above ``pip install`` instruction is compatible with ``conda`` environments. Alternatively, for Linux 64 systems
+once the CUDA driver is correctly set up, you can install CuPy from the ``conda-forge`` channel::
+
+    $ conda install -c conda-forge cupy
+
+and ``conda`` will install pre-built CuPy and most of the optional dependencies for you, including CUDA runtime libraries
+(``cudatoolkit``), NCCL, and cuDNN. It is not necessary to install CUDA Toolkit in advance. If you need to enforce
+the installation of a particular CUDA version (say 10.0) for driver compatibility, you can do::
+
+    $ conda install -c conda-forge cupy cudatoolkit=10.0
+
+.. note::
+
+    Currently cuTENSOR is not yet available on ``conda-forge``.
+
+.. note::
+
+    If you encounter any problem with CuPy from ``conda-forge``, please feel free to report to `cupy-feedstock 
+    <https://github.com/conda-forge/cupy-feedstock/issues>`_, and we will help investigate if it is just a packaging
+    issue in ``conda-forge``'s recipe or a real issue in CuPy.
+
+
```

Comment:
@jakirkham, another question: Do we want to comment on `nvcc`'s availability? Something along this line:

"If you did not install CUDA Toolkit yourselves, the ``nvcc`` compiler might not be available. The ``cudatoolkit`` package does not have ``nvcc`` included."

Reply:
Sure we could do that. It might also make sense to include this info in conda-forge and link out to it. WDYT? 🙂